packages/nimble-blazor/package.json

```diff
@@ -28,6 +28,9 @@
     "type": "git",
     "url": "git+https://github.com/ni/nimble.git"
   },
+  "publishConfig": {
+    "access": "public"
+  },
```

Comment:
this package is weird, we don't publish it (it's not a published npm package, it's a published nuget package) but we don't mark it private so beachball will rev it. notice how the invoke-publish:npm ignores the command above.
So we shouldn't mark this as publicjConfig access public